js/script.js

```diff
@@ -0,0 +1,253 @@
+        console.log(cardStyle);
+
+        if (cardStyle === null) {
+            return;
+        }else {
+            //for each card-text option
+            for (i=0; i<cardTextOption.length; i++) {
+                //if the card style is the same as the card text option
+                if (cardStyle === cardTextOption[i].type) {
+                    $(".card").attr("id","");
+                    //add the card style as a class to the card in the html, linking it to our special styling in our css
+                    $(".card").attr("id", cardStyle);
+                    
+                    //generate a random clever line from our array
+                    var randomLine = cardTextOption[i].lines[Math.floor(Math.random()*cardTextOption[i].lines.length)];
+                    //make a new h4
+                    var cardText = $("<h4>");
+                    //give it the classes card-text and "selected card type" - font
+                    cardText.addClass("card-text");
+                    cardText.addClass(cardTextOption[i].type + "-font")
+                    //assign the random line as the text
+                    cardText.text(randomLine);
+                    
+                    //make a new p tag
+                    var cardSubtext = $("<p>");
+                    //give it the inner text from that card-text option's paragraph 
+                    cardSubtext.text(cardTextOption[i].paragraph);
+
+                    //give the card-title the styled font-class
+                    $(".card-title").addClass(cardTextOption[i].type + "-font")
+
+                    //append the h4 and p tags we made to the card-text-area
+                    $(".card-text-area").append(cardText, cardSubtext);
+
+                }
+            }
+            //unhide the bday card + send-button
+            $(".bday-card").removeClass("hide");
+            $(".send-button").removeClass("hide");
+            
+            $(document).scrollTop($(document).height());
+        }
+    }
+
+    function getAPIImage() {
+        //clear current image on click
+        $(".api-image").empty();
+        //make a variable getting the style choice
+        var cardStyle = $("#card-styles").val();
+        //get the birthday 
+        //user selected month (from dropdown menu) value from 01 - 12
+        var inputMonth = $("#input-month").val();               
+        //user selected day (from dropdown menu)
+        var inputDay = $("#input-day").val();
+        //user input year (typed)
+        var inputYear = $("#input-year").val();
+
+        //make a variable to hold our card image or headlines
+        var cardImage = $("<img>");
+        
+        //make api keys and get functions for each api 
+        //nasa API Function
+        function getNasaImage () {
+            //make query url
+            var nasaURL = "https://api.nasa.gov/planetary/apod?api_key=PxXYhbxjF4qoZihVRPPnN3C4IpvxKF4N1eWiJVRx&date=" +
```

Comment:
this is a very minor thing, but it's a best practice to put the `+` at the beginning of the line

it makes it easier to read and less error prone, you would be surprised how often you end up missing the plus at the end